### PR TITLE
New node for notification

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
Doesnt exists

* **What is the new behavior (if this is a feature change)?**
new node for notication 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This PR is connected to dojot/dojot#977.

* **Other information**:

### CODE REVIEW

File: orchestrator/nodes/notification/notification.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="label.name"></label>
+        <input type="text" id="node-input-name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-edit"></i> <span data-i18n="label.message"></label>
+        <input type="text" id="node-input-message_notification">
+    </div>
+        <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-cube"></i> <span data-i18n="label.source"></label>
```
Comment (raulnegreiros) [inline orchestrator/nodes/notification/notification.html]: `source` refers to `metaAttrsFilter` seems a little confusing. I don't know if it's clear to the user, maybe `metadata` sounds better

File: orchestrator/nodes/notification/notification.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<script type="text/x-red" data-template-name="notification">
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="label.name"></label>
+        <input type="text" id="node-input-name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-edit"></i> <span data-i18n="label.message"></label>
```
Comment (raulnegreiros) [inline orchestrator/nodes/notification/notification.html]: I think is a better idea to use a text box instead of a simple input.

Comment (raulnegreiros) [inline orchestrator/nodes/notification/notification.html]: I think an option to this configuration be read from node configuration or a message's attribute will be useful.

File: orchestrator/nodes/change/index.js [hunk 3]
```diff
@@ -148,7 +148,8 @@ class DataHandler extends dojot.DataHandlerBase {
             case "num":
               this._set(rule.p, Number(rule.to), message);
               break;
-            case "boolean":
+              case "bool":
```
[the remark this answers is not in the document]
Comment (mprevide) [reply orchestrator/nodes/change/index.js]: Yes...

**Pull request merged**